src/components/Label/Label.spec.tsx

```diff
@@ -48,7 +48,7 @@ describe('Label', () => {
         expect(render(<Label variant="success" filled />).container.firstChild).toHaveStyle(`
             color: ${Colors.WHITE};
             border-color: ${Colors.POSITIVE_GREEN_900};
-            backgroundColor: ${Colors.POSITIVE_GREEN_900};
```

Comment:
Was this merged with a failing test or what is going on here?

Reply:
The test wasn't failing, because this property was just ignored (it's not valid css). I fixed it for completeness, so we actually test for the correct background color.